nano/node/active_transactions.cpp

```diff
@@ -1369,6 +1452,9 @@ nano::inactive_cache_status nano::active_transactions::inactive_votes_bootstrap_
 	nano::inactive_cache_status status (previously_a);
 	constexpr unsigned election_start_voters_min{ 5 };
 	nano::uint128_t tally;
+
+	lock_a.unlock ();
```

Comment:
Should probably `debug_assert (lock_a.owns_lock ());` before calling this

Reply:
Updated